### DISCUSSION TITLE
求助运行时的一个

### DISCUSSION THREAD

**Discussion by misefeng** [mikumifa/biliTickerBuy#295] [提问专区]
![图片1(1)](https://github.com/mikumifa/biliTickerBuy/assets/174243793/e4bddee1-323e-4281-9f61-397d77668ce0)
求助一下，今天下午抢BML给弹这个东西，两个号都是这样，请问这是在排队还是什么

**Comment by mikumifa** [comment]
手动验证码，但是你没有操作

**Comment by mikumifa** [comment]
卡在验证码那一步了

**Reply by misefeng** [reply]
感谢感谢，我还以为验证码会自己弹出来的，就没管hh
